test/mld.jl

```diff
@@ -44,6 +44,53 @@
         @test isapprox(result["solution"]["load"]["1"]["status"], 1.000; atol=1e-3)
     end
 
+    @testset "5-bus mn acp mld" begin
+        case5_mn = InfrastructureModels.replicate(case5, 3, Set(["per_unit"]))
+        result = run_mn_mc_mld(case5_mn, ACPPowerModel, ipopt_solver, multinetwork=true)
+
+        @test result["termination_status"] == LOCALLY_SOLVED
+        @test isapprox(result["objective"], 3*0.3377; atol=2.0e-4)
+        @test all(isapprox(result["solution"]["nw"]["1"]["load"]["1"]["pd"], [3.0, 3.0, 3.0]; atol=1.0e-4))
+        @test all(isapprox(result["solution"]["nw"]["2"]["load"]["1"]["qd"], [0.9861, 0.9861, 0.9861]; atol=1.0e-4))
+        @test isapprox(result["solution"]["nw"]["3"]["load"]["1"]["status"], 1.0; atol=1.0e-3)
+    end
+
+    @testset "5-bus mn storage acp mld" begin
+        case5_strg_mn = InfrastructureModels.replicate(case5_strg, 3, Set(["per_unit"]))
+        result = run_mn_mc_mld(case5_strg_mn, ACPPowerModel, ipopt_solver, multinetwork=true)
+
+        @test result["termination_status"] == LOCALLY_SOLVED
+        @test result["objective"] >= 3.0*0.3432
+        @test isapprox(result["solution"]["nw"]["1"]["load"]["1"]["status"], 1.0; atol=1.0e-3)
+    end
+
+    @testset "5-bus mn nfa mld" begin
+        case5_mn = InfrastructureModels.replicate(case5, 3, Set(["per_unit"]))
+        result = run_mn_mc_mld(case5_mn, NFAPowerModel, ipopt_solver, multinetwork=true)
+
+        @test result["termination_status"] == LOCALLY_SOLVED
+        @test isapprox(result["objective"], 3.0*0.1557; atol=2.0e-4)
+        @test isapprox(result["solution"]["nw"]["1"]["load"]["1"]["status"], 1.0; atol=1.0e-3)
+    end
+
+    @testset "5-bus mn storage nfa mld" begin
+        case5_strg_mn = InfrastructureModels.replicate(case5_strg, 3, Set(["per_unit"]))
+        result = run_mn_mc_mld(case5_strg_mn, NFAPowerModel, ipopt_solver, multinetwork=true)
+
+        @test result["termination_status"] == LOCALLY_SOLVED
+        @test result["objective"] >= 3.0*0.1557
+        @test isapprox(result["solution"]["nw"]["1"]["load"]["1"]["status"], 1.0; atol=1.0e-3)
+    end
+
+    @testset "5-bus mn lpubfdiag mld" begin
+        case5_mn = InfrastructureModels.replicate(case5, 3, Set(["per_unit"]))
+        result = run_mn_mc_mld(case5_mn, LPUBFDiagPowerModel, ipopt_solver, multinetwork=true)
+
+        @test result["termination_status"] == LOCALLY_SOLVED
+        @test isapprox(result["objective"], 3.0*0.1557; atol=2.0e-4)
+        @test isapprox(result["solution"]["nw"]["1"]["load"]["1"]["status"], 1.0; atol=1.0e-3)
+    end
```

Comment:
You provide an implementation in the IV space but not a test.  A test should be added or the IV formulation should be removed.

Reply:
There's an implementation in the IV space for OPF but not MLD. We can talk about if we need an IV implementation.